### PR TITLE
Update badge URLs

### PR DESCRIPTION
**Pull request by shinnn**
This repo has been transferred to https://github.com/image-size/image-size.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,10 +1,8 @@
...
-[![NPM Downloads](http://img.shields.io/npm/dm/image-size.svg)](https://npmjs.org/package/image-size)
+[![Build Status](https://travis-ci.org/image-size/image-size.svg?branch=master)](https://travis-ci.org/image-size/image-size)
+[![NPM Downloads](https://img.shields.io/npm/dm/image-size.svg)](https://www.npmjs.com/package/image-size)
 [![Gittip](http://img.shields.io/gittip/netroy.svg)](https://www.gittip.com/netroy/)
-
-[![Coverage Status](https://coveralls.io/repos/netroy/image-size/badge.png?branch=master)](https://coveralls.io/r/netroy/image-size?branch=master)
-[![Code Climate](https://codeclimate.com/github/netroy/image-size.png)](https://codeclimate.com/github/netroy/image-size)
-[![Dependency Status](https://gemnasium.com/netroy/image-size.png)](https://gemnasium.com/netroy/image-size)
+[![Coverage Status](https://coveralls.io/repos/netroy/image-size/badge.svg?branch=master&service=github)](https://coveralls.io/github/netroy/image-size?branch=master)
+[![devDependency Status](https://david-dm.org/image-size/image-size/dev-status.svg)](https://david-dm.org/image-size/image-size#info=devDependencies)
```
Comment (shinnn) [inline Readme.md]: I've switched URL of the dependency badge from https://gemnasium.com/ to faster https://david-dm.org/.

**Comment by shinnn** [comment]
@netroy @zeke Can you re-enable Coveralls integration? Currently https://coveralls.io/github/image-size/image-size doesn't work.

File: Readme.md [hunk 2]
```diff
@@ -1,10 +1,8 @@
-[![Build Status](https://travis-ci.org/netroy/image-size.png?branch=master)](https://travis-ci.org/netroy/image-size)
-[![NPM Downloads](http://img.shields.io/npm/dm/image-size.svg)](https://npmjs.org/package/image-size)
+[![Build Status](https://travis-ci.org/image-size/image-size.svg?branch=master)](https://travis-ci.org/image-size/image-size)
+[![NPM Downloads](https://img.shields.io/npm/dm/image-size.svg)](https://www.npmjs.com/package/image-size)
 [![Gittip](http://img.shields.io/gittip/netroy.svg)](https://www.gittip.com/netroy/)
```
Comment (netroy) [inline Readme.md]: can you remove this badge please?

Comment (shinnn) [inline Readme.md]: Will fix soon.

Comment (shinnn) [inline Readme.md]: https://github.com/image-size/image-size/commit/3c7fde9055e04591e3522d0c25459c810d63d41e

**Pull request merged**